api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -183,44 +183,162 @@ message JwtHeader {
   string value_prefix = 2;
 }
 
-// This is the Envoy HTTP filter config for JWT authentication.
-// [#not-implemented-hide:]
-message JwtAuthentication {
-  // List of JWT rules to valide.
-  repeated JwtRule rules = 1;
+// This message specifies a Jwt requirement.
+message RequiredProvider {
+  // Specify the issuer for the required JwtProvider.
+  string issuer = 1;
 
-  // If true, the request is allowed if JWT is missing or JWT verification fails.
-  // Default is false, a request without JWT or failed JWT verification is not allowed.
-  bool allow_missing_or_failed = 2;
+  // This audiences field overrides the one in JwtProvider if not empty.
```

Comment:
What is meant here by overriding? E.g. does this make the match more specialized than if just using the JwtProvider?

Reply:
Semantic wise, audiences specified in the requirement will override the ones specified in the Provider.   
@llchan do you know how such audience override can be used?